src/store/plugins/names.js

```diff
@@ -116,7 +112,10 @@ export default (store) => {
         commit,
       }) {
         commit('setAreNamesFetching', true);
-        const fetchPendingNameClaimTransactions = (address) => fetchPendingTransactions(address)
+        const { fetchPendingTransactions } = ProtocolAdapterFactory.getAdapter(PROTOCOL_AETERNITY);
+        const fetchPendingNameClaimTransactions = (address) => (
+          fetchPendingTransactions(address)
+        )
```

Comment:
It makes no sense to create separate function that is a shortcut to another. This can be removed and in the following code just use:
```
adapter.fetchPendingTransactions(address);
```